music/demos/ddsp_tone_transfer.ts

```diff
@@ -0,0 +1,201 @@
+/**
+ * @license
+ * Copyright 2020 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {AudioFeatures, ModelValues} from '../src/ddsp/interfaces';
+import * as mm from '../src/index';
+import {SPICE} from '../src/index';
+
+enum MODEL {
+  VIOLIN = 'violin',
+  TENOR_SAXOPHONE = 'tenor_saxophone',
+  TRUMPET = 'trumpet',
+  FLUTE = 'flute',
+}
+
+export const PRESET_MODEL_URL =
+    'https://storage.googleapis.com/magentadata/js/checkpoints/ddsp';
+
+export const PRESET_MODELS = {
```

Comment:
I don't think you really need this separate array. Since you use it as `PRESET_MODELS[MODEL.VIOLIN]`, that value is exactly just `${PRESET_MODEL_URL}/${MODEL.TRUMPET}`. Like, you're not getting anything from the array itself.

Reply:
good point. this array had some settings before, but you're right that it doesn't make sense to keep it without those settings now.